src/http/vault.ts

```diff
@@ -91,12 +78,6 @@ export class VaultClient extends JsonRpcClient {
         await this.post("withdraw_collateral", [request.toHex()]);
     }
 
-    async updateBtcAddress(network: bitcoin.Network): Promise<string> {
```

Comment:
Also this one should stay for now.

Reply:
This functionality has been removed in the vault client. If required, we could add the calls for `updatePublicKey` and `registerAddress`.

Reply:
@nud3l Should I add `updatePublicKey` and `registerAddress` now or in a future PR?

Reply:
We would also need to add these calls to the `vault` client as well...

Reply:
Hm, easier of course if we can just remove this. How is this working on replace now @gregdhill? Do we derive an address for the vault as well or would the vault want to update their btc address inc ase they want to be replaced?

Reply:
If the vault accepts a replace request it generates a new address via Bitcoin Core and submits that via the `accept_replace` call.

Reply:
Ok, then this is good to be removed since we don't need to update the address. @savudani8 can you please create a ticket on the UI board just to make sure that we remove that from the UI as well?